### PR TITLE
Partners homepage section (#76)

### PR DESCRIPTION
**Pull request by rossbruniges**
Working on #76 - new logos and all

### CODE REVIEW

File: gameon/static_site/templates/static_site/landing.html [hunk 1]
```diff
@@ -33,10 +33,30 @@
         <h2 class="exclaim">Submit your game prototype by {{ request.challenge.end_date.strftime('%B %e, %Y') }}</h2>
         <ul>
             <li class="make whimper">Make web-based hackable games, games for learning or cross-platform games.</li>
-            <li class="invent whimper"><a href="#">Attend a Game Jam near you [link through to gameon event page when merged in]</a> or host your own! Our <a href="https://webmaker.org/events/guides/game-jam/">handy Event Kits</a> make it easy.</li>
+            <li class="invent whimper"><a href="{{ url('events.list') }}">Attend a Game Jam near you</a> or host your own! Our <a href="https://webmaker.org/events/guides/game-jam/">handy Event Kits</a> make it easy.</li>
             <li class="win whimper">Win a trip to the <a href="http://www.gdconf.com/">Game Developers Conference</a> to meet industry pioneers.</li>
             <li class="market whimper">Have your completed game featured in the <a href="http://www.mozilla.org/en-US/apps/">Firefox Marketplace</a> and published on <a href="http://www.chillingo.com/">Chilingo</a>.</li>
```
Comment (andrewhayward) [inline gameon/static_site/templates/static_site/landing.html:38]: Not an addition, granted, but should this link really point to a localised location?

(This link being `http://www.mozilla.org/en-US/apps/`)

File: gameon/base/static/base/css/home.css [hunk 2]
```diff
@@ -113,6 +113,39 @@ div[role=main] .big-picture .busta {
...
+}
+.partners img {
+    width: 100%;
+    height: auto;
+}
+.with li {
+    width: 25%;
+    
+}
+.with .lg {
```
Comment (andrewhayward) [inline gameon/base/static/base/css/home.css:143]: Does this mean 'large'?

File: gameon/base/static/base/css/home.css [hunk 3]
```diff
@@ -113,6 +113,39 @@ div[role=main] .big-picture .busta {
...
+    width: 35%;
+}
+.with .cl {
```
Comment (andrewhayward) [inline gameon/base/static/base/css/home.css:146]: Does this mean 'clear'?

**Comment by andrewhayward** [comment]
Other than the minor comments, looks good to me.

File: gameon/static_site/templates/static_site/landing.html (hunk 1 above)
Comment (rossbruniges) [inline gameon/static_site/templates/static_site/landing.html:38]: File a new bug for this - as I've noticed there are a couple of other links that point to the US version

File: gameon/base/static/base/css/home.css (hunk 2 above)
Comment (rossbruniges) [inline gameon/base/static/base/css/home.css:143]: Actually means 'long' - but either way I couldn't face using such a horrid class name :(

File: gameon/base/static/base/css/home.css (hunk 3 above)
Comment (rossbruniges) [inline gameon/base/static/base/css/home.css:146]: Yes - but either way I couldn't face using such a horrid class name :(

File: gameon/base/static/base/css/home.css (hunk 2 above)
Comment (rossbruniges) [inline gameon/base/static/base/css/home.css:143]: I'll sort this when I have time to tidy up the CSS task

File: gameon/base/static/base/css/home.css (hunk 3 above)
Comment (rossbruniges) [inline gameon/base/static/base/css/home.css:146]: I'll sort this when I have time to tidy up the CSS task

**Pull request merged**